### PR TITLE
Update to 1.20

### PR DESCRIPTION
**Pull request by JRoy**
- [x] Bump build and supported versions to 1.20.1
  - [x] Spigot API
  - [x] `run-server`
  - [x] Readme
  - [x] `VersionUtil`
- [x] Add new items to `items.json`
- [x] Add new entity types to `/spawnmob`
  - [x] Add camel and sniffer to `Mob`
  - [x] Add `camel:saddle` to `MobData`/`MobCompat` (if not already covered by SADDLE_HORSE?)
- [x] Add support for new sign types and reverse side
  - [x] Allow new sign types to be used as command signs (`MaterialUtil#isSign` etc.)
  - [x] Decide on how command signs will use the reverse side
  - [x] Update `/editsign` to support the reverse side

### CODE REVIEW

**Review by tadhgboyle — APPROVED** [review]

**Review by LaxWasHere — APPROVED** [review]

**Comment by joaoespanhol** [comment]
i need update 1.20

**Review by tanzbr — COMMENTED** [review]
good

**Review by iContinux — APPROVED** [review]

**Review by NekrosBurek — APPROVED** [review]

**Review by torchsmith — APPROVED** [review]

**Comment by littlekan233** [comment]
support versions doesn't have 1.20, only 1.20.1

**Comment by JRoy** [comment]
> support versions doesn't have 1.20, only 1.20.1

that's correct, we only support the latest patch version of an mc release

**Comment by dandud100** [comment]
So if this gets merged the 2.20.1 release will be build or am i misunderstanding this?

**Comment by Evidentsinger14** [comment]
When this is merged, a dev build for 2.20.1 will be published, yes.

**Review by littlekan233 — CHANGES_REQUESTED** [review]
add 1.20 support plz qwq

**Comment by torchsmith** [comment]
Hi @mdcfe do you have any estimate on when you'll be able to get done the items.json task?

**Comment by 0kju0** [comment]
Please merge this early.

**Review by pop4959 — DISMISSED** [review]

File: Essentials/src/main/resources/items.json [hunk 1]
```diff
@@ -5467,7 +6172,7 @@
   "begg": "dragon_egg",
   "bossegg": "dragon_egg",
   "degg": "dragon_egg",
-  "dragonegg": "dragon_egg",
```
Comment (mdcfe) [inline Essentials/src/main/resources/items.json]: aaaaaaaa

File: Essentials/src/main/resources/items.json [hunk 2]
```diff
@@ -41089,7 +42252,7 @@
   "wither_skeletonsegg": "wither_skeleton_spawn_egg",
   "wither_skeletonspawn": "wither_skeleton_spawn_egg",
   "wither_skeletonspawnegg": "wither_skeleton_spawn_egg",
-  "withersegg": "wither_skeleton_spawn_egg",
+  "withersegg": "wither_spawn_egg",
```
Comment (mdcfe) [inline Essentials/src/main/resources/items.json]: aaaaaaaaaaaa

**Review by mdcfe — DISMISSED** [review]

File: EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/AccountLinkManager.java [hunk 3]
```diff
@@ -68,12 +68,7 @@ public UUID getUUID(final String discordId) {
     public boolean unlinkAccount(InteractionMember member) {
         Preconditions.checkNotNull(member, "member cannot be null");
 
-        if (!isLinked(member.getId())) {
-            return false;
-        }
-
-        removeAccount(member, DiscordLinkStatusChangeEvent.Cause.UNSYNC_API);
-        return true;
+        return isLinked(member.getId()) && removeAccount(member, DiscordLinkStatusChangeEvent.Cause.UNSYNC_API);
```
Comment (mdcfe) [inline EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/AccountLinkManager.java]: Uh

**Review by mdcfe — APPROVED** [review]

**Pull request merged**